Library/ViewModels/PaymentMethodsViewModel.swift

```diff
@@ -37,9 +37,9 @@ PaymentMethodsViewModelInputs, PaymentMethodsViewModelOutputs {
     self.reloadData = self.viewDidLoadProperty.signal
 
     let paymentMethodsEvent = Signal.merge(
-      self.viewWillAppearProperty.signal,
```

Comment:
I'm wondering if we should remove the `viewWillAppear` input signal completely and switch the tracking call to `viewDidLoad` or if tracking is intentionally done on `viewWillAppear` 🤔

Reply:
Good catch!

We actually talked about this yesterday with Justin (should have maybe reposted on Slack - sry about that)

Currently the tracking behaviour acts differently on iPhone and iPad devices (this is due to the way iPad lifecycle events are called for certain modal presentation styles - specifically the one we use `.pageSheet`?). It honestly should be fine (this is how the iOS was intended to work).

We can definitely switch to `viewDidLoad` or think about different ways to track these events but I’d suggest we do this thorough the whole app since this is not the only place that’s affected.

Reply:
☝️☝️☝️ Cc @Scollaco @cdolm92 for more transparency